dev/com.ibm.websphere.security.wim.base/resources/com/ibm/websphere/security/wim/util/resources/WebsphereWimUtilMessages.nlsprops

```diff
@@ -18,6 +18,15 @@
 
 # Message prefix block: Profile Manager: 1000-1999
 
+
+DUPLICATE_PROPERTY_EXTENDED=CWIML1996W: The extended property ''{0}'' has already been defined for the ''{1}'' entity and will be ignored.
+DUPLICATE_PROPERTY_EXTENDED.explanation=The extended property has already been defined and will be ignored as there cannot be two properties with the same name.
+DUPLICATE_PROPERTY_EXTENDED.useraction=Check that there are not multiple definitions for the extended property in the server configuration and delete or rename duplicates.
+
+DUPLICATE_PROPERTY_ENTITY=CWIML1997W: The extended property ''{0}'' cannot override the property of the same name for the ''{1}'' entity and will be ignored.
```

Comment:
DUPLICATE_PROPERTY_ENTITY=CWIML1997W: The extended property ''{0}'' is ignored because it cannot override the property of the same name for the ''{1}'' entity.

Reply:
I can do this as well. Will wait for your response on the first message before delivering.